crowdin.yml

```diff
@@ -1,4 +1,7 @@
 commit_message: '[ci skip]'
+skip_untranslated_strings: true
+skip_untranslated_files: true
```

Comment:
Maybe this shouldn't be added, since the i18n checks still want a file, even if it is empty